ui/app/components/alert-popup.js

```diff
@@ -8,14 +6,22 @@ import OuterHTML from './outer-html';
  * // All properties are passed in from the flashMessage service.
  *   <AlertPopup @type={{message-types flash.type}} @message={{flash.message}} @close={{close}}/>```
  *
- * @param type=null {String} - The alert type. This comes from the message-types helper.
- * @param [message=null] {String} - The alert message.
- * @param close=null {Func} - The close action which will close the alert.
+ * @param {string} type=null - The alert type. This comes from the message-types helper.
+ * @param {string} [message=null] - The alert message.
+ * @param {function} close=null - The close action which will close the alert.
```

Comment:
I noticed `@isPreformatted` in the template file, should we add it here as a param?

Reply:
Good call. I just copied over the original documentation, and didn't check if it covered everything. Nice catch.